SlamTests/Slam/ShowQueueViewControllerSpec.swift

```diff
@@ -29,5 +32,44 @@ class ShowQueueViewControllerSpec: QuickSpec {
                 expect(queueManager.queueView).to(beIdenticalTo(controller.queue))
             }
         }
+
+        describe("clicking the refresh queue button") {
+            var refreshQueueButton = UIBarButtonItem()
+            var addMatchToQueueButton = UIBarButtonItem()
+            var tableView = UITableView()
+            var controller = ShowQueueViewController()
+            var manager = MockQueueManager()
```

Comment:
This seems to be a lot of dependencies. Let's plan on talking about a way to simplify this part of the design next Friday. (5/8)